### PR TITLE
Features/add hbase db connector

### PR DESCRIPTION
**Pull request by Wu-Jing**
- [x] Change columns parameter from string to actual json
- [x] Add tests
- [x] Add examples to Activities.md

### CODE REVIEW

File: connectors/hbase-connector/src/test/scala/test/yumi/pipeline/activities/connectors/hbase/HBaseWriterActivitySpec.scala [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    val resultSessionData = activity.invoke(sessionData)
+
+    // Assert
+    verify(dataFrameWriter).write(
+      dataFrame = dataFrame,
+      format = "org.apache.spark.sql.execution.datasources.hbase",
+      mode = mode,
+      path = "org.apache.spark.sql.execution.datasources.hbase",
+      options = expectedOptions)
+
```
Comment (coderdiaries) [inline connectors/hbase-connector/src/test/scala/test/yumi/pipeline/activities/connectors/hbase/HBaseWriterActivitySpec.scala]: Extraline here.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -12,7 +12,11 @@ object Dependencies {
   val mySqlVersion = "5.1.12"
   val postgreSqlVersion = "9.1-901-1.jdbc4"
   val sqlServerVersion = "6.1.0.jre8"
+  val shcVersion = "1.1.0-2.1-s_2.11"
+  val hBaseSparkVersion = "2.0.0-SNAPSHOT"
```
Comment (coderdiaries) [inline project/Dependencies.scala]: Lines 15 and 16 not used.

File: connectors/hbase-connector/src/test/scala/test/yumi/pipeline/activities/connectors/hbase/HBaseWriterActivitySpec.scala (hunk 1 above)
Comment (Wu-Jing) [inline connectors/hbase-connector/src/test/scala/test/yumi/pipeline/activities/connectors/hbase/HBaseWriterActivitySpec.scala]: Removed

File: project/Dependencies.scala (hunk 2 above)
Comment (Wu-Jing) [inline project/Dependencies.scala]: Removed line 16, I think 15 is used

**Pull request merged**